scripts/debcheck.sh

```
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Should it use `set -e` to fail early?
```suggestion
#!/bin/bash

set -e
```

---

and also pipefail, yes!